nums/models/glms.py

```diff
@@ -350,10 +382,6 @@ def hessian(self, X: BlockArray, y: BlockArray, mu: BlockArray = None):
 PoissonRegressor = PoissonRegression
 
 
-def line_search():
```

Comment:
is line search now part of nums/models/lbfgs.py (BackTrackingLineSearch)?

Reply:
Yes.